xmtp_mls/src/groups/mod.rs

```diff
@@ -236,40 +329,72 @@ where
                     sender_installation_id,
                     sender_wallet_address: sender_account_address,
                 };
-                message.store(&mut self.client.store.conn()?)?;
+                message.store(conn)?;
             }
             ProcessedMessageContent::ProposalMessage(_proposal_ptr) => {
                 // intentionally left blank.
             }
             ProcessedMessageContent::ExternalJoinProposalMessage(_external_proposal_ptr) => {
                 // intentionally left blank.
             }
-            ProcessedMessageContent::StagedCommitMessage(_commit_ptr) => {
-                // intentionally left blank.
+            ProcessedMessageContent::StagedCommitMessage(staged_commit) => {
+                println!("[{}] received staged commit", self.client.account_address());
+                openmls_group.merge_staged_commit(provider, *staged_commit)?;
```

Comment:
I added this line to make my test case handling conflicting messages produce the right errors